includes/class-boldgrid-editor-assets.php

```diff
@@ -163,6 +163,31 @@ public static function is_webpack() {
 		return ( defined( 'BGEDITOR_SCRIPT_DEBUG' ) && BGEDITOR_SCRIPT_DEBUG );
 	}
 
+	/**
+	 * Maybe Enqueue Animate.css
+	 *
+	 * Checks if post uses the animate.css or not.
+	 *
+	 * @since SINCEVERSION
+	 *
+	 * @return bool True if animate.css is to be enqueued.
+	 */
+	public function maybe_enqueue_animate() {
+		global $post;
+
+		if ( ! $post ) {
+			return true;
+		}
+
+		$post_content = apply_filters( 'the_content', $post->post_content );
+
+		if ( preg_match( '/class=".*wow.*"/', $post_content ) ) {
```

Comment:
This could affect lots of sites. Do you think we should write a quick PHPUnit test to throw a few article at it and ensure it's matching successfully?

Reply:
I used regex101.com to test the regex, and make sure that it matches as desired.